### PR TITLE
Remove machine only from rules that are applicable to containers

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Remove platform machine only from rules that are applicable to container.

#### Rationale:

- These rules are applicable to containers therefore can't be marked as machine only.

- Address raised issues on this PR: https://github.com/ComplianceAsCode/content/pull/4240

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml [hunk 1]
```diff
@@ -11,5 +11,3 @@ description: |-
     In Red Hat Enterprise Linux servers and workstations, hardware token login
     {{% endif %}}
     is not enabled by default and must be enabled in the system settings.
-
-platform: machine
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml]: The group smart_card_login conatins rule service_pcscd_enabled, which uses systemd check but OpenSCAP cannot evaluate systemd checks in containers.

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml]: Agreed. Rule reverted to as machine only.

Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml]: > The group smart_card_login conatins rule service_pcscd_enabled, which uses systemd check but OpenSCAP cannot evaluate systemd checks in containers.

This is something that needs to be fixed, btw.

**Comment by matusmarhefka** [comment]
ACK

**Pull request merged**